crates/tauri-bundler/src/bundle/windows/nsis/mod.rs

```diff
@@ -140,16 +140,20 @@ fn add_build_number_if_needed(version_str: &str) -> anyhow::Result<String> {
         "{}.{}.{}.{}",
         version.major, version.minor, version.patch, version.build
       ));
-    } else {
-      anyhow::bail!("optional build metadata in app version must be numeric-only");
     }
   }
 
+  log::warn!(
+    "Unable to parse version build metadata. Numeric value expected, received: {}",
```

Comment:
```suggestion
    "Unable to parse version build metadata. Numeric value expected, received: `{}`. This will be replaced with `0` in `VIProductVersion` because Windows requires this field to be numeric.",
```